### PR TITLE
Portfolio: Added a contact form and finished the mobile version

### PR DESCRIPTION
**Pull request by roniy68**
# Highlights  🥇 

- Added Contact Form in the footer
- Added Fields ***[Name, Email, Messsage and a button]*** **with the correct requirement fields**
- Added social media icons 
- Added Formspree Service
- Used Flexbox
- Implemented Button Interaction

uu

# Followed Rules: 📏 
- Completed this project with coding partner @Eugeneiyukhun

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -264,3 +266,100 @@ button {
...
 }
+
+/* Form Section */
+
+#form-session {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: column;
+  margin-top: 50px;
+}
```
Comment (Yazino12) [inline css/style.css:277]: - [x] Flexbox is used ✔

File: index.html [hunk 2]
```diff
@@ -196,6 +196,40 @@ <h2 class="project-title">Skills</h2>
...
+                <div class="container">
+                  <h1 class="supporting-text">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                  </h1>
+                  <form action="https://formspree.io/f/mjvznanj" method="post" id="form">
+                        <input type="text" id="name" placeholder="Full name" required maxlength="30" name="name">
+                        <input type="email" id="email" placeholder="Email address" required name="email">
+                        <textarea name="" id="message" cols="30" rows="10" placeholder="Enter Text Here" required maxlength="500" name="message"></textarea>
+                        <button>Get in touch</button>
+                  </form>
```
Comment (Yazino12) [inline index.html:210]: - [ ] Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html) ✔.
- [ ] All input fields are wrapped in a single `form` element ✔.
- [ ] All input fields have the correct associated placeholder text and type attribute ✔.

**Review by Yazino12 — APPROVED** [review]
## Hi @roniy68 and @Eugeneiyukhun 🖐🏾,

You guys have done an outstanding job implementing the form 🎯, and there is nothing else left to say other than it's time to merge it 🚀🚀.

### To Highlight:
- Descriptive pull request title and summary ✔
- Correct use of GitHub flow ✔
- Organized code ✔

# STATUS: APPROVED ✅✅✅
CLAP
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
- **_[OPTIONAL]_** In your future projects please use [this template](https://github.com/microverseinc/readme-template) and remember that all sections are mandatory except the [FAQ](https://github.com/microverseinc/readme-template#:~:text=%E2%9D%93-,FAQ,-Add%20at%20least) in order to present your project with as much detail as possible 🤝🏽.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (roniy68) [reply css/style.css:277]: 😃 Thanks

**Comment by roniy68** [comment]
# Thanks For your Review

@Yazino12 

**Merging the branch**

merge

**Pull request merged**